### PR TITLE
Feature/10575 accessibility and functional test publish in release pipeline

### PR DESCRIPTION
**Pull request by amogh14893**
Changes in Azure-pipelines.yml for moving Accessibility to Deployment Stage.
Changes in jest.config for Accessibility Tests and Functional Tests to publish results in junit.xml

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -89,19 +89,16 @@ stages:
             inputs:
               testRunner: 'JUnit'
               testResultsFiles: 'junit.xml'
+              testRunTitle: Unit Test Results
             displayName: 'Publish Unit Test Results'
 
+
```
Comment (ravi102465) [inline azure-pipelines.yml:95]: Can we remove extra line

File: azure-pipelines.yml [hunk 2]
```diff
@@ -228,6 +251,49 @@ stages:
               npm install -D jest jest-playwright-preset playwright
              displayName: 'Install Jest Playwright Dependencies'
              
+           - task: DownloadBuildArtifacts@1
```
Comment (ravi102465) [inline azure-pipelines.yml:254]: Remove this task we don't need it

File: azure-pipelines.yml [hunk 3]
```diff
@@ -228,6 +251,49 @@ stages:
...
+             inputs:
+                buildType: 'current'
+                downloadType: 'single'
+                artifactName: 'AccessibilityArtifact'
+                downloadPath: '$(Build.SourcesDirectory)'
+             displayName: Download Accessibility Test Artifact
+
+           - task: DownloadBuildArtifacts@1
```
Comment (ravi102465) [inline azure-pipelines.yml:262]: Remove this task we don't need it

**Review by ravi102465 — APPROVED** [review]
Looks good to me once comment are resolved

File: azure-pipelines.yml (hunk 3 above)
Comment (amogh14893) [reply azure-pipelines.yml]: Done the change as requested

File: azure-pipelines.yml (hunk 2 above)
Comment (amogh14893) [reply azure-pipelines.yml]: Done the change as requested

File: azure-pipelines.yml (hunk 1 above)
Comment (amogh14893) [reply azure-pipelines.yml]: Done the change as requested

**Comment by ravi102465** [comment]
closing this PR changes were duplicate with other branch and it is in main

**Pull request closed without merging**